sites/platform/src/learn/tutorials/migrating.md

```diff
@@ -33,6 +33,7 @@ and for some apps, such as Drupal, configuration that you need to export from th
 ## 2. Create a project
```

Comment:
@chadwcarlson I'm sorry but I didn't manage to vendorize steps 2 and 3 on this page :(.